File: ILT.IHR.APP/ClientApp/src/app/core/interfaces/ProcessData.ts
import { IProcessDataTicketDisplay } from "./ProcessDataTicket";


export class IProcessData {
    ProcessDataID: number;
    ProcessWizardID: number;
    Process?: string;
    Data?: string;
    DataColumns?: string;
    StatusId?: number;
    Status?: string;
    ChangeNotificationEmailId?: string;
    EmailApprovalValidity?: number;
    ProcessedDate?: Date;
    CreatedBy: string;
    CreatedDate: Date;
    ModifiedBy?: string;
    ModifiedDate?: Date;
}
export class IProcessDataDisplay {
    processDataID: number;
    processWizardID: number;
    process?: string;
    data?: any;
    dataColumns?: string;
    statusId: number;
    status?: string;
    changeNotificationEmailId?: string;
    emailApprovalValidity: number;
    processedDate: Date;
    createdBy: string;
    createdDate: Date;
    modifiedBy: string;
    modifiedDate: Date;
    timeStamp: ['']
    processDataTickets: any[];
}

File: ILT.IHR.DTO/ProcessData.cs
using System;
using System.Collections.Generic;
using System.Text;
namespace ILT.IHR.DTO
{
    public partial class ProcessData : AbstractDataObject
    {
        public int ProcessDataID
        {
            get { return base.RecordID; }
            set { base.RecordID = value; }
        }
        public int ProcessWizardID { get; set; }
        public string Process { get; set; }
        public string Data { get; set; }      
        public string DataColumns { get; set; }
        public int StatusId { get; set; }
        public string Status { get; set; }
        public string ChangeNotificationEmailId { get; set; }
        public int EmailApprovalValidity { get; set; }
        public DateTime? ProcessedDate { get; set; }
        public List<ProcessDataTicket> ProcessDataTickets { get; set; }

    }
}
